### DISCUSSION TITLE
How to install ulauncher on GNU/Gentoo?

### DISCUSSION THREAD

**Discussion by Shwen-Y** [Ulauncher/Ulauncher#1137] [Q&A]
How to install ulauncher on GNU/Gentoo?

中文：如何在GNU/Gentoo上安装ulauncher？

**Comment by friday** [comment]
I know nothing about Gentoo, but you can possibly check the build instructions for Arch and "translate" those: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ulauncher (stable v5) or https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=ulauncher-git (unreleased wip v6 branch)
